docs/git-workflow.md

```diff
@@ -45,48 +76,48 @@ If the development mode is automated, turn off the automatic "merge", since you
 
 Perform the final round of testing on `narrative-dev`:
 
-* Work in the `staging` branch, making bugfixes directly on this branch
+- Work in the `staging` branch, making bugfixes directly on this branch
 
         git checkout staging
         # bugfixes and testing
 
-* Test it.
-* Seriously, test it. This includes "stress" and "acceptance" tests.
- 
-Once you are satisfied that the `staging` branch is fit for production, you should commit any 
+- Test it.
+- Seriously, test it. This includes "stress" and "acceptance" tests.
+
+Once you are satisfied that the `staging` branch is fit for production, you should commit any
 changes you made:
 
     git commit -a # .. or whatever
     git push
 
 Now you will create the new release in `master`.
 
-* Merge the staging branch into the master branch.
+- Merge the staging branch into the master branch.
 
         git checkout master; git merge staging
 
-* Tag the master branch (using current version number).
+- Tag the master branch (using current version number).
 
         ver=`python -m biokbase.narrative.__init__`
         git tag -a "v$ver" -m "release version $ver"
 
-* Push the `master` branch, with tags, up to github
+- Push the `master` branch, with tags, up to github
 
         git push origin "v$ver"
 
-* Switch to the production host. Deploy on `narrative` using the newly created tag.
+- Switch to the production host. Deploy on `narrative` using the newly created tag.
```

Comment:
Can devs even do this?! I thought this was a DevOps task.

(Not your content, I know)

Reply:
We'll find out today at Jason's demo. I think that might be part of the plan - making a tag would build a production image. But I think it'll take a devops person to push The Button that'll do the actual deployment.

Reply:
Ya, I have no idea. I think updating this part of the documentation will be a separate JIRA ticket.